tools/integration-tests/tester/tests/drng_test.go

```diff
@@ -0,0 +1,45 @@
+package tests
+
+import (
+	"encoding/hex"
+	"log"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestDRNG checks whether drng messages are actually relayed/gossiped through the network
+// by checking the messages' existence on all nodes after a cool down.
+func TestDRNG(t *testing.T) {
+	drng, err := f.CreateDRNGNetwork("TestDRNG", 5, 8)
+	require.NoError(t, err)
+	defer drng.Shutdown()
+
+	// wait for randomness generation to be started
+	log.Printf("Waiting for randomness generation to be started...\n")
+	time.Sleep(100 * time.Second)
+
+	peers := len(drng.Peers())
+	randomnessMap := make(map[string]int)
+
+	// check for randomness on every peer
+	for i := 0; i < 3; i++ {
+		for _, peer := range drng.Peers() {
+			resp, err := peer.GetRandomness()
+			require.NoError(t, err)
+			log.Println(resp)
```

Comment:
This should use `t.Log` and use the `String()` function or extract the parameters you want/need to see to avoid a cryptic output like this: 
```2020/05/06 13:49:11 &{7 2020-05-06 13:49:11.030803713 +0000 UTC [191 58 172 186 178 90 43 238 162 122 248 194 214 41 24 132 189 41 200 84 8 42 129 210 168 182 144 176 17 14 80 185 176 171 79 130 188 245 43 195 168 221 210 24 213 131 127 110 79 115 190 192 233 139 136 152 214 66 25 89 85 145 56 24] }```